src/ui/components/vurdering-av-livets-sluttfase-form/VurderingAvLivetsSluttfaseForm.tsx

```diff
@@ -83,6 +98,20 @@ const VurderingAvLivetsSluttfaseForm = ({
         }
     };
 
+    const perioderSomBlirVurdert = formMethods.watch(FieldName.PERIODER);
+    const harVurdertAlleDagerSomSkalVurderes = React.useMemo(() => {
+        const dagerSomSkalVurderes = (resterendeVurderingsperioder || []).flatMap((p) => p.asListOfDays());
+        const dagerSomBlirVurdert = (perioderSomBlirVurdert || [])
+            .map((period) => {
+                if ((period as AnyType).period) {
```

Comment:
as AnyType? dette så litt hacky ut

Reply:
Den er kopiert fra de andre skjemaene. Men jeg antar det er for å unngå explicit-any type check feil. Og så er den generalisert på toppen for å slippe flere eslint-disable i koden. Kanskje?